scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -260,6 +260,14 @@ abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanne
                             "and $scannerCriteria, not scanning the package again $packageIndex."
                 }
 
+                storedResults.forEach {
+                    if (!archiver.hasArchive(pkg.id.toPath() + "/" + it.provenance.hash())) {
```

Comment:
`"${pkg.id.toPath()}/${it.provenance.hash()}"` for consistency. We rarely use string concatenation.